### DISCUSSION TITLE
v0.66.0

### DISCUSSION THREAD

**Discussion by tommoor** [outline/outline#4058] [Announcements]
> **Note** This update requires database migrations, take your installation offline, backup the database, and run with yarn db:migrate or yarn db:migrate --env production-ssl-disabled
>
> Once the migrations have run, subscription data can be (optionally) backfilled with the following command – this will create a notification subscription to each document that users have contributed to:
> `node build/server/scripts/20220722000000-backfill-subscriptions.js`

### Enhancements

* External SSO methods now allow logging into teams as long as emails match by @thenanyu in https://github.com/outline/outline/pull/3813
* Viewers can now be upgraded to editors on individual collections or groups in https://github.com/outline/outline/pull/4023
* It's now possible to subscribe to a document without editing it, and also to unsubscribe from documents you have by @CuriousCorrelation in https://github.com/outline/outline/pull/3834
* New application-wide request limiter disabled by default. See [documentation](https://wiki.generaloutline.com/share/770a97da-13e5-401e-9f8a-37949c19f97e/doc/rate-limiter-HSqErsUgXH).
* Added an emailed confirmation code to account deletion in https://github.com/outline/outline/pull/3873
* A random color is now chosen on collection creation by @apoorv-mishra in https://github.com/outline/outline/pull/3912
* Support for Google Form embeds by @apoorv-mishra in https://github.com/outline/outline/pull/3930
* Notifications for export completing are now optional by @codemicro in https://github.com/outline/outline/pull/3935
* Added support for Grist embeds. by @prio in https://github.com/outline/outline/pull/3914
* The draw.io integration now supports self-hosted deployments by @apoorv-mishra in https://github.com/outline/outline/pull/3980

### Fixes

* Fixed authentication error the maximum number of teams has been reached in https://github.com/outline/outline/pull/3819
* Use previous language when creating code block from markdown shortcut in https://github.com/outline/outline/pull/3830
* Cannot create collection if all existing collections are deleted in https://github.com/outline/outline/pull/3836
* Long collection description prevents import in https://github.com/outline/outline/pull/3847
* substitution of content when sending an image to a profile in https://github.com/outline/outline/pull/3869
* Cleanup attachments uploaded to storage when import fails in https://github.com/outline/outline/pull/3868
* Expand highlighted languages by @spotlightishere in https://github.com/outline/outline/pull/3891
* Improve document delete confirmation message by @apoorv-mishra in https://github.com/outline/outline/pull/3876
* Harden regex for embeds by escaping by @apoorv-mishra in https://github.com/outline/outline/pull/3907
* Removed templatize action for trashed document by @apoorv-mishra in https://github.com/outline/outline/pull/3922
* refactor auth flow to explicitly pass in a host by @thenanyu in https://github.com/outline/outline/pull/3909
* Put request rate limit at application server by @apoorv-mishra in https://github.com/outline/outline/pull/3857
* Upgrade `markdown-it` to fix text collapse bug in https://github.com/outline/outline/pull/3953
* chore: Rate limiter audit in https://github.com/outline/outline/pull/3965
* perf: Suppress Mermaid diagram rendering when hidden in https://github.com/outline/outline/pull/3963
* Reference email image by cid for self hosted instances in https://github.com/outline/outline/pull/3957
* perf: Refactoring socket event payloads in https://github.com/outline/outline/pull/4011
* Content is displayed wrongly when printing / Save as PDF by @Nicicalu in https://github.com/outline/outline/pull/4043
* Remove ability to use `GET` for RPC API requests by default in https://github.com/outline/outline/pull/4042
* Allow backlinks to work with fully qualified urls and anchors in https://github.com/outline/outline/pull/4050
* Login screen not vertically centered on mobile in https://github.com/outline/outline/pull/4052
* Handle GitLab can be configured for tokens to not expire. in https://github.com/outline/outline/pull/4051
* Post-signin redirect path is no longer saved in https://github.com/outline/outline/pull/4054

## New Contributors
* @apoorv-mishra made their first contribution in https://github.com/outline/outline/pull/3841
* @spotlightishere made their first contribution in https://github.com/outline/outline/pull/3891
* @psmoros made their first contribution in https://github.com/outline/outline/pull/3906
* @codemicro made their first contribution in https://github.com/outline/outline/pull/3935
* @prio made their first contribution in https://github.com/outline/outline/pull/3914
* @Nicicalu made their first contribution in https://github.com/outline/outline/pull/4043

**Full Changelog**: https://github.com/outline/outline/compare/v0.65.1...v0.66.0

<hr /><em>This discussion was created from the release <a href='https://github.com/outline/outline/releases/tag/v0.66.0'>v0.66.0</a>.</em>

**Comment by Nicicalu** [comment]
I don't know if I'm the only one, but the newest docker image doesn't seem to be the right version:

When I pull the newest image through `docker-compose` it seems to be the same as the last version.
I tried to pull with `:0.66.0` and `:latest` and both times there are no pending migrations and when  checking in settings it's says it's version `0.65.2`.
The added features are missing aswell.

Anyone else having this problem?

**Reply by ChetanGoti** [reply]
I just upgraded couple of Outline instances using latest `0.66.0` Docker image, and all have been upgraded correctly along with migrations. 

The container images are differs as well.

```bash
ubuntu@chetan:~/apps/outline$ docker images | grep outline
outlinewiki/outline                         0.66.0            6444993f4e43   8 hours ago    479MB
outlinewiki/outline                         0.65.2            6d55b35b2c46   6 weeks ago    466MB
```

**Reply by Nicicalu** [reply]
@ChetanGoti 
Thanks for the quick reply. I had to delete the volume of the container. My container was mounted to the disk, for some tests I did, so that didn't update properly.
Works now.
